src/Components/ReusableComponents/Header/Header.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Link, NavLink } from 'react-router-dom';
+import './Header.css';
+import PropTypes from 'prop-types';
+
+const Header = () => {
```

Comment:
Later on we can add some conditional logic outside of the render to only show the relevant navigation links, but that's not a priority whatsoever.